src/main/java/JavaTwo/LabTwo/MainTest.java

```diff
@@ -8,23 +8,27 @@
 
 public class MainTest {
 
-
-    public static void main(String[] args) throws IOException {
```

Comment:
I believe it wasn't a proper way to handle it, whole responsibility is put on the main method (and then on jvm)